Source/WebKit/Shared/ApplePay/ios/WebPaymentCoordinatorProxyIOS.mm

```diff
@@ -52,7 +52,8 @@
 #if HAVE(PASSKIT_DISBURSEMENTS)
     std::optional<ApplePayDisbursementPaymentRequest> webDisbursementPaymentRequest = request.disbursementPaymentRequest();
     if (webDisbursementPaymentRequest) {
-        auto disbursementRequest = platformDisbursementPaymentRequest(request, originatingURL);
+        auto disbursementRequest = platformDisbursementPaymentRequest(request, originatingURL, webDisbursementPaymentRequest->requiredRecipientContactFields);
+        // TODO: get webPageProxyID etc added
```

Comment:
This is no longer relevant, so you can drop this comment!

Reply:
argh bad rebase, sorry about that!